native/SampleApps/RestAPIExplorer/RestAPIExplorer.xcodeproj/project.pbxproj

```diff
@@ -944,6 +944,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				BUNDLE_PREFIX = com.salesforce.mobilesdk;
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
This is different: it's actually ensuring that the project will use/expect modules where necessary.  The other setting we've been squashing is the `CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES` setting, which provides a stop gap from fixing underlying module errors.